test/framework/sandbox/easybuild/easyblocks/generic/bar.py

```diff
@@ -32,6 +32,18 @@
 from easybuild.framework.easyconfig import CUSTOM, MANDATORY
 
 
+class dummy1:
+    """Only to verify that unrelated classes in software specific easyblocks are ignored"""
+
+
+class dummy2(dummy1):
+    """Same but with inheritance"""
+
+
+class dummy1:
```

Comment:
Should probably be dummy3 to not confuse anyone who reads this code.

Reply:
```suggestion
class dummy3:
```